docs/sources/tempo/configuration/ipv6.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configure IPv6
+weight: 37
+---
+
+# Configure IPv6
+
+Tempo can be configured to communicate between the components using Internet Protocol Version 6, or IPv6.
```

Comment:
Which components? Do you mean the ingester, compactor, etc.? (I'm assuming but it might be useful to be more specific so it's clear to a reader whether this is IPv6 between Tempo architecture pieces or Tempo to outside network, or all of it.)

Reply:
Its communication between all tempo components, which is a little vague, but if we change communication patterns later, then its still vague but true.

Reply:
Should I clarify and say something about "all tempo components"?